server/seeds/accounts.js

```diff
@@ -39,8 +39,8 @@ Meteor.startup(function() {
     }
 
     //add additional collections to houston admin
-    Houston.add_collection(Meteor.users);
-    Houston.add_collection(Houston._admins);
+    // Houston.add_collection(Meteor.users);
```

Comment:
Hey do we still use Houston? If so, would be nice to have ability to add admins and manage users.